doc/subject.md

```diff
@@ -69,7 +69,7 @@ There are also a few specializations of the `Subject` type: `BehaviorSubject`, `
 
 ## Multicasted Observables
 
-Whenever we refer to "a multicasted Observable", that means an Observable execution that passes through a Subject. Otherwise, a "plain (unicast) Observable" behaves like an *invokable collection of future values*, with no sharing assumed.
+A "multicasted Observable" passes notifications through a Subject which may have many subscribers, whereas a plain "unicast Observable" only sends notifications to a single Observer.
```

Comment:
Fair enough, however `behaves like an *invokable collection of future values*, with no sharing assumed` is definitely confusing and out of place here. "Plain" observables have already had a lot literature up to this point, and this sentence doesn't help to make the distinction between multicast which is also a collection of future values.